src/components/Forms/LoginForm/LoginForm.tsx

```diff
@@ -24,6 +25,20 @@ export const LoginForm = ({ setPageType }) => {
   const handleSubmit = async (event) => {
     event.preventDefault();
 
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
Very good naming convention, it's immediately clear that current Regex dealing with emails, that's very helpful.

Reply:
Moved validation logic to separate file in utils folder